op-chain-ops/cmd/celo-migrate/README.md

```diff
@@ -0,0 +1,79 @@
+# Celo L2 Migration Script
+
+## Overview
+
+This script has two main sections that were originally separate scripts. One migrates Celo blocks to a format compatible with `op-geth`, while the other performs necessary state changes (TODO: add more info on state changes here).
```

Comment:
@palango What can we say here to summarize what the state migration is needed for?

Reply:
Nevermind, I added a section on this. Let me know if anything looks off